ts/components/session/SessionPasswordModal.tsx

```diff
@@ -31,6 +31,17 @@ export class SessionPasswordModal extends React.Component<Props, State> {
 
     this.setPassword = this.setPassword.bind(this);
     this.closeDialog = this.closeDialog.bind(this);
+
+    this.onKeyUp = this.onKeyUp.bind(this);
+    window.addEventListener('keyup', this.onKeyUp);
+  }
+
+  public componentDidMount() {
+    setTimeout(() => $('#password-modal-input').focus(), 100);
+  }
```

Comment:
Looks very hacky to me. No way to make it work with refs and without the setTimeout?  something like this: https://stackoverflow.com/a/28890330

Reply:
Doesnt work with our current implementation.